dspace/config/submission-forms.xml

```diff
@@ -531,18 +531,18 @@
             </row>
             <row>
                 <relation-field>
-                    <relationship-type>isProjectOfOrgUnit</relationship-type>
+                    <relationship-type>isOrgUnitOfProject</relationship-type>
                     <search-configuration>orgunit</search-configuration>
                     <repeatable>false</repeatable>
                     <label>Organization</label>
                     <hint>Enter the organization's name</hint>
                     <linked-metadata-field>
-                        <dc-schema>project</dc-schema>
-                        <dc-element>funder</dc-element>
-                        <dc-qualifier>name</dc-qualifier>
+                        <dc-schema>dc</dc-schema>
+                        <dc-element>contributor</dc-element>
```

Comment:
I don't understand this change. Why, in the context of a project, do you consider a funder as a dc.contributor.other?

If you check https://schema.org/ResearchProject you will find the funder there. So this change doesn't make sense to me.

Reply:
Is not a funder, it's the linked orgunit in the context of the project.
This relationship is mapped by the type `isOrgUnitOfProject` you can find it also inside the `virtual-metadata.xml` configuration, here:  https://github.com/DSpace/DSpace/blob/main/dspace/config/spring/api/virtual-metadata.xml#L100-L102.